delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -706,6 +691,36 @@ def list_port_groups(self, storage_id):
             LOG.error("Failed to get port groups  from msa")
             raise e
 
+    @staticmethod
+    def get_port_group_id_and_name(port_number, storage_port_list):
+        native_port_group_id = None
+        native_port_group_name = None
+        if port_number:
+            port_codes = port_number.split(',')
+            for port_code in port_codes:
+                if storage_port_list:
```

Comment:
if 'storage_port_list' is a list, no need to add 'if storage_port_list'

Reply:
Delete judgment if 'storage_port_list'